code/modules/projectiles/guns/energy/laser.dm

```diff
@@ -21,6 +21,7 @@
 /obj/item/gun/energy/laser/retro
 	name ="retro laser gun"
 	icon_state = "retro"
+	item_state = "retro"
```

Comment:
```suggestion
```
Can be left out since both are the same. It'll take the `icon_state` if `item_state` is undefined.
Also goes for the others

Reply:
@farie82 I removed these lines as you suggested and loaded up a test server and the in-hand sprites reverted back to the old laser sprite.

I think it is because it is still inheriting `item_state = "laser"` from the gun/energy/laser parent, which is why this doesn't work.

Many of the other guns use the following to get around this:
`item_state = null`

I will keep things as is for now, since everything works, but if a change is needed for consistency or code formatting, please let me know. (for example, using `item_state = null` instead.)